stdlib/sys.mlp

```diff
@@ -45,6 +46,10 @@ let unix = unix ()
 let win32 = win32 ()
 let cygwin = cygwin ()
 let max_array_length = max_wosize ()
+let max_float_array_length =
+  if word_size = 64 || get_flat_float_array ()
```

Comment:
We should leave this computation to the external primitive, so that alternative runtimes such as js_of_ocaml could specify a different value.